### DISCUSSION TITLE
Tiptap vue2/3 and Vuetify with long content becomes slow

### DISCUSSION THREAD

**Discussion by PeterSchoell** [ueberdosis/tiptap#2509] [General Questions]
I use Tiptap in a vue project. With very long texts the input slows down a lot. 
The example https://tiptap.dev/examples/book runs very smooth without Vuetify - but very slow with Vuetify. 
I have tested both vue2 and vue3 with Vuetify 3. 

Input becomes extremely slow when the text contains only one paragraph and this one paragraph contains about 10,000 words (generated via https://www.blindtextgenerator.de/) 

Does anyone have an idea what this could be?

**Comment by jakedolan** [comment]
A code sandbox would be useful to help determine what might be causing the poor performance with how you use it combination with Vuetify.

I have no experience with Vuetify specifically, but I can give some of my experience of testing long texts (like Moby Dick in the example). I found performance issues in two places:

1. The getHTML() function was very slow (hundreds of ms). To work around this, I debounced the @input emit from the onUpdate hook. (versus using v-model)
2. The isActive() function was also causing issues and there are some workarounds though I see that you are aware of these. https://github.com/ueberdosis/tiptap/issues/1930

Hopefully this may be helpful to you or others happening upon the same questions.

**Reply by PeterSchoell** [reply]
I have created a project in the codesandbox. https://codesandbox.io/s/epic-ride-g540f?file=/src/plugins/vuetify.js
Once the CSS(import 'vuetify/dist/vuetify.min.css')  is removed, the editor runs fast. With CSS, the editor is impossible to use with this content. 

The adjustments you suggested (https://github.com/ueberdosis/tiptap/issues/1930) , have no effect here yet. In general, however, these changes are necessary and I have implemented them in my real project.

**Reply by philippkuehn** [reply]
`.v-application { display: flex }` seems to be the issue. I think it’s a browser bug in webkit/blink because it’s not an issue in firefox.
